modules/nw-ptp-e810-hardware-configuration-reference.adoc

```diff
@@ -4,7 +4,7 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="nw-ptp-wpc-hardware-pins-reference_{context}"]
-= Intel Westport Channel E810 hardware configuration reference
+= Intel E810 NIC hardware configuration reference
 
 Use this information to understand how to use the link:https://github.com/openshift/linuxptp-daemon/blob/release-4.16/addons/intel/e810.go[Intel E810-XXVDA4T hardware plugin] to configure the E810 network interface as PTP grandmaster clock.
```

Comment:
Is the hardware plugin really referenced as E810-XXVDA4T? Plugin itself just called `e810`

Reply:
Updated it to "E810 hardware plugin"